README.md

```diff
@@ -290,6 +291,7 @@ Options for the `Builder` class
     * `xmldec.standalone` standalone document declaration: true or false
   * `doctype` (default `null`): optional DTD. Eg. `{'ext': 'hello.dtd'}`
   * `headless` (default: `false`): omit the XML header. Added in 0.4.3.
+  * `cdata` (default: `false`): wrap text nodes in `<![CDATA[ ... ]]>` 
```

Comment:
> I am not sure it makes sense to wrap everything in CDATA vs. just parts that would be escaped otherwise. Opinions on this?

Hmm, interesting. I guess we could try adding a normal text node, then if it fails (throws an error), we can fallback to trying with a CDATA wrap?

> say which version this was added.

Do you want me to bump the version as part of this PR?